CONTRIBUTING.md

```diff
@@ -273,6 +273,22 @@ The `-pip` key should be removed when the package becomes available on all platf
 As a reminder `pip` rules should not be used on Gentoo.
 See above for the Gentoo specific guidelines.
 
+#### pip mixed with system packages
+
+Some packages are only available as system packages in newer distributions.
+In such case, you can specify the default as system package, and pip for older distributions.
```

Comment:
```suggestion
In such case, you can specify the default as system package, and pip for older distributions for which the system packages are not available.
```

Reply:
At first I thought it was a bit too descriptive. But after passing through auto-translators, I got the idea. Thanks!